routes.js

```diff
@@ -60,6 +67,24 @@ var apiLimiter = rateLimit({
   }
 });
 
+var waitCaptchaMin = isDev ? 1: 120;
+var captchaLimiter = rateLimit({
+  store: (isDev ? undefined : new MongoStore({
+    uri: 'mongodb://127.0.0.1:27017/captchaLimiter',
+    resetExpireDateOnChange: true, // Rolling
+    expireTimeMs: waitCaptchaMin * 60 * 1000 // n minutes for mongo store
+  })),
+  windowMs: waitCaptchaMin * 60 * 1000, // n minutes for all stores
+  max: 1, // limit each IP to n requests per windowMs for memory store or expireTimeMs for mongo store
+  handler: function (aReq, aRes, aNext, aOptions) {
+    aRes.type('svg').status(200).send(
+      svgCaptcha('429 Too Many Requests', Object.assign(settings.captchaOpts, {
```

Comment:
@jesus2099 

> Is it about the captcha problem when editing profile?

Yes. Now look at this line of code first. 

I don't expect everyone to read the code changes but a search for it would have been nice from you before posting accidental misinformation on production. **ALL** 4xx codes are user error and server error codes are 5xx or something alpha like ENOENT.